src/lib/components/header.svelte

```diff
@@ -21,7 +21,10 @@
 
   export let username: string | undefined;
 
-  const navItems = [{ name: 'Docs', path: '/docs' }];
+  const navItems = [
+    { name: 'Docs', path: '/docs' },
+    { name: 'AppPage', path: '/app' },
```

Comment:
Shall we just call it `App` or something? I'm not sure about this camle case here?